usaspending_api/bulk_download/v2/views.py

```diff
@@ -171,31 +208,37 @@ def post(self, request):
 
         if not agency_id:
             # Return all the agencies if no agency id provided
-            response_data['agencies'] = toptier_agencies
+            cfo_agencies = sorted(list(filter(lambda agency: agency['cgac_code'] in CFO_CGACS,
+                                              toptier_agencies)),
+                                  key=lambda agency: CFO_CGACS.index(agency['cgac_code']))
+            other_agencies = sorted([agency for agency in toptier_agencies
+                                     if agency not in cfo_agencies],
+                                    key=lambda agency: agency['name'])
+            response_data['agencies'] = {"cfo_agencies": cfo_agencies,
+                                         "other_agencies": other_agencies}
         else:
             # Get the top tier agency object based on the agency id provided
             top_tier_agency = list(filter(lambda toptier: toptier['toptier_agency_id'] == agency_id, toptier_agencies))
             if not top_tier_agency:
                 raise InvalidParameterException('Agency ID not found')
             top_tier_agency = top_tier_agency[0]
             # Get the sub agencies and federal accounts associated with that top tier agency
-            response_data['sub_agencies'] = Agency.objects \
-                .filter(toptier_agency_id=agency_id) \
-                .values(subtier_agency_name=F('subtier_agency__name'),
-                        subtier_agency_id=F('subtier_agency__subtier_agency_id'))
-
-            response_data['federal_accounts'] = FederalAccount.objects \
-                .filter(agency_identifier=top_tier_agency['cgac_code']) \
-                .values(federal_account_name=F('account_title'),
-                        federal_account_id=F('id'))
+            response_data['sub_agencies'] = sorted(Agency.objects
```

Comment:
Wouldn't it be better to order in Postgres?

Reply:
Yes, definitely. I'll try to get this in before merging.